funcx_websocket_service/connection.py

```diff
@@ -0,0 +1,51 @@
+import asyncio
+import logging
+import time
+
+logger = logging.getLogger(__name__)
+
+
+class WebSocketConnection:
+    """A WebSocket connection that has important WebSocket session properties
+    for a single connected client, along with holding the client's WebSocket
+    object which is created by the server
+    """
+
+    def __init__(self, ws):
```

Comment:
I would recommend adding the timeout as a kwarg here, rather than hardcoding it in `check_idle`